tests/dummy/app/controllers/demo/select.js

```diff
@@ -66,6 +82,11 @@ export default Ember.Controller.extend({
 
   vegToppings: Ember.computed('toppings.[]', function() {
```

Comment:
Regarding the computed.filterBy, the search should not be case sensitive so that won't work.